SIT.Manager.Avalonia/Views/InstallPage.axaml

```diff
@@ -17,7 +17,7 @@
 		</Style>
 	</UserControl.Styles>
 					
-	<Grid ColumnDefinitions="Auto, *">
+	<Grid ColumnDefinitions="Auto, *" VerticalAlignment="Top" Margin="0,5,0,0">
```

Comment:
Not vibing with the vertical alignment being at the top instead of center

Reply:
When you extend the window to full screen it looks crappy at least for me. But will remove it.